common/src/main/java/com/hammy275/immersivemc/client/config/screen/ItemGuideCustomizeScreen.java

```diff
@@ -136,11 +136,30 @@ protected void init() {
                         (newMode) -> {
                             ImmersiveMCConfig.placementGuideMode.set(newMode.ordinal());
                             ImmersiveMCConfig.placementGuideMode.save();
-                            ActiveConfig.loadConfigFromFile();
+                            // this is unneeded as it overwrites other guide values that were changed
+                            // in the same active config instance
+                            // ActiveConfig.loadConfigFromFile();
+                            ActiveConfig.placementGuideMode = newMode; // this should be used instead
```

Comment:
Please no commented out code. Instead, you could rework the comment to state that we intentionally don't reload the whole config here to prevent other changed settings from resetting themselves. Lastly, please also remove the "this should be used instead" comment.